user.py

```diff
@@ -12,3 +14,16 @@
 
 class User:
     __name__ = 'nereid.user'
+
+    use_twofactor_auth = fields.Boolean('Use Two-Factor Auth?')
```

Comment:
Should just read `'Use Two-Factor Auth'` (no question mark) IMHO, as checkbox already implies it's a boolean field.